winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -713,16 +514,43 @@ static SECURITY_STATUS SEC_ENTRY kerberos_InitializeSecurityContextA(
 	if (output_tok.length != 0)
 	{
 		CopyMemory(output_buffer->pvBuffer, output_tok.value, output_tok.length);
-		sspi_gss_release_buffer(&(context->minor_status), &output_tok);
+		sspi_gss_release_buffer(&minor, &output_tok);
 	}
 
-	if (context->major_status & SSPI_GSS_S_CONTINUE_NEEDED)
+	if (major & SSPI_GSS_S_CONTINUE_NEEDED)
 		return SEC_I_CONTINUE_NEEDED;
 
-	if (SSPI_GSS_ERROR(context->major_status))
+	if (SSPI_GSS_ERROR(major))
+	{
+		gss_log_status_messages(major, minor);
 		return SEC_E_INTERNAL_ERROR;
+	}
 
 	return SEC_E_OK;
+#else
+	return SEC_E_INTERNAL_ERROR;
```

Comment:
Not your code, but don't we have a `not implemented` return code (something like function not implemented could do the job)

Reply:
I did in fact write that (diffs can get confusing) but you're right; I'll change that